interoperator/pkg/constants/constants.go

```diff
@@ -2,18 +2,25 @@ package constants
 
 // Constants used by interoperator
 const (
-	LeaderElectionID = "interoperator-leader-election-helper"
 	FinalizerName    = "interoperator.servicefabrik.io"
 	ErrorCountKey    = "interoperator.servicefabrik.io/error"
 	LastOperationKey = "interoperator.servicefabrik.io/lastoperation"
 	ErrorThreshold   = 10
 
-	ConfigMapName   = "interoperator-config"
-	ConfigMapKey    = "config"
-	NamespaceEnvKey = "POD_NAMESPACE"
+	ConfigMapName      = "interoperator-config"
+	ConfigMapKey       = "config"
+	NamespaceEnvKey    = "POD_NAMESPACE"
+	OwnClusterIDEnvKey = "CLUSTER_ID"
+	MasterClusterID    = "1"
```

Comment:
So, do we have the constraint that master cluster name has to be `1` ?

Reply:
Default scheduler was hard cording cluster id as "1". Updated it to use from the constant. So now we dont need it to be "1".

Reply:
Done